src/coercions/string.rs

```diff
@@ -25,3 +25,10 @@ impl ToRust<String> for CheckedValue<String> {
         unsafe { std::str::from_utf8_unchecked(slice) }.to_string()
     }
 }
+
+impl ToRuby for String {
+    fn to_ruby(self) -> VALUE {
+        let cstr = CString::new(self).unwrap();
+        unsafe { sys::rb_str_new_cstr(cstr.as_ptr()) }
```

Comment:
Would it make sense to use [`rb_utf8_str_new`](https://github.com/ruby/ruby/blob/b493d156de6506c52222296bf0c26256d0f0479e/string.c#L727) or similar since we know the length and encoding?